packages/parser/tests/unit/schema/appsync.test.ts

```diff
@@ -87,24 +151,24 @@ describe('AppSync Resolver Schemas', () => {
       ]);
     });
 
-    it.each(table)('$name', ({ filename }) => {
-      const event = getTestEvent<AppSyncResolverEvent>({
-        eventsPath,
-        filename,
-      });
-
+    it('should parse resolver event without identity field', () => {
+      const event: Omit<AppSyncResolverEvent, 'identity'> = omit(
+        ['identity'],
+        appSyncResolverEvent
+      );
       const parsedEvent = AppSyncResolverSchema.parse(event);
+      expect(parsedEvent).toEqual(event);
+    });
 
+    it.each(table)('$name', ({ event }: Table) => {
```

Comment:
Let's move most of the sentence in the test header and use the parameter for the part that changes, so it's easier to know what the test group does. I.e. `should parse resolver event with $case`

Reply:
*done